Source/BSN.Commons.Orm.EntityFrameworkCore/RepositoryBase.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Commons.Infrastructure;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+
+namespace BSN.Commons.Orm.EntityFrameworkCore
+{
+    public class RepositoryBase<T> : global::Commons.Infrastructure.IRepository<T> where T : class
```

Comment:
Please add API comment or ref to the interface. (if IRepository have API comments)

Reply:
Add API commnets for all public  class.